starky/src/fibonacci_stark.rs

```diff
@@ -81,6 +81,10 @@ impl<F: RichField + Extendable<D>, const D: usize> Stark<F, D> for FibonacciStar
     ) {
         todo!()
     }
+
+    fn constraint_degree(&self) -> usize {
+        2
```

Comment:
Seems like it's degree 1, but I guess that would give a quotient degree of 0 which wouldn't work? How about if we did

```rust
fn quotient_degree_factor(&self) {
    1.max(self.constraint_degree() - 1)
}
```

Reply:
(or maybe a global method so it's not public and can't be overridden)

Reply:
> Seems like it's degree 1, but I guess that would give a quotient degree of 0 which wouldn't work?

I think it's of degree 2. For example 
```rust
yield_constr.constraint_first_row(vars.local_values[0] - vars.public_inputs[Self::PI_INDEX_X0]);
```
has degree 2.

Reply:
Oh yeah makes sense.